Formula/node.rb

```diff
@@ -49,6 +49,8 @@ def install
 
     bootstrap = buildpath/"npm_bootstrap"
     bootstrap.install resource("npm")
+    # These dirs must exists before npm install.
+    mkdir_p libexec/"lib"
```

Comment:
Should I add this to https://github.com/Homebrew/brew/blob/24ae318a3d5f0519f5fe29d08e01e8a044d99d87/Library/Homebrew/language/node.rb#L39-L60, as the same issue happened for `bcoin`?